### ISSUE TITLE
Update to copy on Repositories step

### ISSUE THREAD

**Issue by dkriethof** [eclipse-pass/main#744]
Per Laurel:

"I am in the staging site pretending I'm a person without a grant but with an article to deposit. When I get to the Repositories step, the language erroneously indicates that I must deposit into JScholarship, which is not true. I can, but I don't have to. I could deposit the work elsewhere, host it on my own site, or publish it OA by paying an APC. The JHU OA policy is publishing-model-agnostic.

We should change the language to read something like: "Based on the grant and journal information provided, JScholarship could help you fulfill your obligation to make your work open access, per the JHU Open Access policy."

![Image](https://github.com/eclipse-pass/main/assets/135133952/774e2d58-f0fd-4d47-8535-184dd12a6bbd)

**Comment by jaredgalanis** [comment]
waiting for working group feedback

**Comment by dkriethof** [comment]
I _think_ what Laurel was saying is if you submit something that is not going to PMC (and just JScholarship), the language isn't 100% correct since the Open Access policy at JHU is more of a "strong suggestion" than a requirement. But will wait to hear back from the WG on that.

**Comment by jaredgalanis** [comment]
Makes sense. The only other thing I think we should be clear on is that whatever the open access policy says strong suggestion/totally optional/required, it's still 100% necessary to select _some_ repository to continue in this flow. Laurel was only seeing this language because there was only 1 repository that matched and therefore that 1 repository is actually required.

**Comment by dkriethof** [comment]
Ah, I see. I think the difference is you are thinking of the workflow language while she is thinking of just the overall Open Access policy. Maybe we can tweak it to make both things clear?

**Comment by markpatton** [comment]
Copying in a comment, because I think something is getting lost which Jared alludes to.

What happens is that pass divides repositories into required and optional. If there are some required repositories from grants for example, then the institutional repo (jscholarship) is listed as optional. When there is only one repo then jscholarship is listed as required. This makes sense from a logical perspective, well what are you doing here if not depositing? But when you then read the text about the required repository above it seems to say things about jscholarship which are actually not true as the ticket points out. So one solution is to let jscholarship be listed as optional even if there is only one repo. Then the ui will have to be updated to not let them click the next button unless a repo is selected.

**Comment by dkriethof** [comment]
@jaredgalanis @markpatton Looks like the copy now reads "Based on the information you provided so far, these are the public access policies that are applicable to your work"

![Image](https://github.com/eclipse-pass/main/assets/135133952/cdb2ddec-4737-4796-9e37-4d85576f74f1)

I think this seems clearer. Should we just close this ticket?
